.github/workflows/tests.yml

```diff
@@ -33,6 +33,21 @@ jobs:
       - run: brew test-bot --only-formulae
         if: github.event_name == 'pull_request'
 
+      - name: Rename bottle
+        if: github.event_name == 'pull_request'
+        run: |
+          autoload zmv
+          if [[ "${ImageOS}" = macos15 ]]; then
+            (zmv 'mas-(*).arm64_sequoia.bottle.tar.gz' 'mas-$1.arm64_big_sur.bottle.tar.gz') || true
+          else
+            (zmv 'mas-(*).ventura.bottle.tar.gz' 'mas-$1.high_sierra.bottle.tar.gz') || true
+          fi
+
+      - name: Download 1.8.6 bottle
+        if: github.event_name == 'pull_request' && matrix.os == 'macos-15'
+        run: |
+          curl -LO https://github.com/mas-cli/mas/releases/download/v1.8.6/mas-1.8.6.el_capitan.bottle.tar.gz
```

Comment:
This step has version `1.8.6` hardcoded in it. Is this meant to test something about the new release?

Reply:
No. 1.8.6 is the newest version of mas that support macOS 10.11 & 10.12. 1.8.7+ only supports 10.13+. Those breaking changes were made years ago when swift was bumped to 5.7.1.

Reply:
I don't know if including 1.8.6 bottles will work. We'll test this in mas prod with a beta release. If anything goes wrong with this, I'll fix it.

Reply:
Ah, gotcha.